_summaries/gropp2020implicit_2.md

```diff
@@ -1,19 +1,85 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+paper: gropp2020implicit
+author: # samatharhay
+score: # 7
 ---
 
-TODO: Summarize the paper:
 * What is the core idea?
+
+The core idea of the paper was to introduce a new paradigm built on implicit geometric regularization for computing high fidelity implicit neural representations directly from raw data scans.
+
 * How is it realized (technically)?
+
+Given an input cloud X = {x_i}iEI C R^3 with or without normal data, N = {n_i} iEI C R^3, the goal is to compute the parameters \theta of a multi-layer perceptron (MLP) f(x;\theta), where f has the shape R^3 x R^m \rightarrow R, such that it approximates a signed distance function to a plausible surface M defined by X and N.
+
+To help learn these parameters the authors propose the loss function:
+
+![image of function 2](gropp2020implicit_2a.png)
+
+where \lambda > 0 is a hyperparameter, ||.|| = ||.||2 (the euclidean 2-norm) and 
+
+![image of function 3](gropp2020implicit_2b.png)
+
+which encourages f to vanish on X.
+
+The second term in the first equation is called the Eikonal term, which encourages the gradients to be of unit 2-norm.
+
+Optimizing the loss function using SGD, or a variant of SGD) results in solutions that are close to a signed distance function with a smooth and suprising ly plausible zero level set. 
```

Comment:
surprising ly --> surprisingly

Consider mentioning the Implicit Geometrical Regularization property here as a consequence of the plane reproduction property. Also that the plane reproduction property means the loss function will always arrive at a minima when optimizing with SGD or variants.

Then tell the readers to read the proof of plane reproduction in the paper if interested.